src/locales/en.json

```diff
@@ -382,16 +373,16 @@
     "country_select": "Select country"
   },
   "passport": "Passport",
-  "driving_licence": "Driver's License",
+  "driving_licence": "Driver's license",
   "national_identity_card": "Identity Card",
   "short_passport": "passport",
   "short_driving_licence": "license",
   "short_national_identity_card": "card",
-  "bank_building_society_statement": "Bank/Building Society Statement",
+  "bank_building_society_statement": "Bank or building society statement",
   "utility_bill": "Utility Bill",
   "benefit_letters": "Benefits Letter",
   "government_letter": "Government Letter",
-  "council_tax": "Council Tax Letter",
+  "council_tax": "Council tax letter",
```

Comment:
This title is inconsistent with the other ones that have capitalisation. Could you update the spreadsheet with `Council Tax Letter` please?

Reply:
fixed